.github/pull_request_template.md

```diff
@@ -0,0 +1,17 @@
+## Description
+_describe the intent of your changes here_
+
+
+#### GitHub Issue: [Closes|Fixes|Resolves] #_your GitHub issue number here_
+
+### Changes
+* ...
+* ...
+* ...
+
+### Testing Strategy
+_describe how you or someone else can test and verify the changes_
+
+
+### Concerns
+_describe any concerns that might be worth mentioning or discussing_
```

Comment:
We should add a definition of done with items that need to be verified before creating a PR. Example: Test with reference implementation of ambi.